lib/tasks/temp.rake

```diff
@@ -1,6 +1,14 @@
 # -*- coding: utf-8 -*-
 namespace :temp do
 
+  desc 'Populate public_token column of InfoRequest'
+  task populate_public_token: :environment do
```

Comment:
I've just noticed that request pages error without running this, and it looks like it may take a while to run:

```ruby
info_request = InfoRequest.find(101)

single_request_time = Benchmark.realtime {  
  public_token = InfoRequest.public_token_from_id(info_request.id)
  info_request.update_column(:public_token, public_token)
}

total_time_in_minutes = single_request_time * 600_000
# => 8641.321817412972
```

I think we should ensure the request pages load successfully with a `NULL` `public_token`, and only enable the feature if one exists.

Reply:
Fixed in e7f4edc8181aefb4e522f107e6cf7672b8d30de1